debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCObjectHeapIteratorAddressOrderedList_V1.java

```diff
@@ -75,9 +80,10 @@ protected GCObjectHeapIteratorAddressOrderedList_V1(U8Pointer base, U8Pointer to
 						excludedRangeList.add(new U8Pointer[] {heapAlloc, heapTop});
 					} else {
 						/* Might be an instrumented VM */
-						U8Pointer realHeapTop = adjustedToRange(vmThread.allocateThreadLocalHeap().realHeapTop(), base, top);
-						if(realHeapTop.notNull() && isSomethingToAdd(heapAlloc, realHeapTop)) {
-							excludedRangeList.add(new U8Pointer[] {heapAlloc, realHeapTop});
+						U8Pointer realHeapAlloc = getRealHeapAlloc(vmThread.allocateThreadLocalHeap(), heapAlloc, base, top);
+						U8Pointer realHeapTop = getRealHeapTop(vmThread.allocateThreadLocalHeap(), heapTop, base, top);
```

Comment:
Why does this need both `realHeapAlloc` and `realHeapTop`? Only one will be present except in a very small number of interim builds that I think we can safely ignore.

Reply:
we fake heapAlloc for disabling inline cache in old core file and fake heapTop in new core file, in case we scan the TLH for gccheck during disabling inline TLH, we need to skip scanning between realHeapAlloc to heapTop for old core and skip scanning between heapAlloc and realHeapTop for new core, in order to make most code transparent between old core(V1) and new core(V2), getRealHeapAlloc() and getRealHeapTop() return realHeapAlloc and HeapTop in V1 and return heapAlloc and realHeapTop in V2.